spec/indexers/descriptive_metadata/pub_year_1e_spec.rb

```diff
@@ -570,8 +570,8 @@
         }
       end
 
-      xit 'selects date with status primary' do
-        expect(doc).to include('sw_pub_date_facet_ssi' => '2021-04-02')
+      it 'selects date with status primary' do
+        expect(doc).to include('sw_pub_date_facet_ssi' => '2022')
```

Comment:
@arcadiafalcone this field is only an integer year, so I modified these few tests at the bottom.  Do they still look okay?

Reply:
Yes, these are all correct.